src/averageAges.js

```diff
@@ -15,11 +22,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter((person) => person.sex === 'm');
+
+  return century === undefined
+    ? averageAgeCalculator(men)
+    : averageAgeCalculator(
+      men.filter((person) => Math.ceil(person.died / 100) === century)
+    );
```

Comment:
In your code, the check for century === undefined is actually redundant, because in the case where century is undefined, it will return century without filtering. You can simplify the code by removing this check.
return century
  ? averageAgeCalculator(
      men.filter((person) => Math.ceil(person.died / 100) === century)
    )
  : averageAgeCalculator(men);